### ISSUE TITLE
Version 1.1.0 causes my server to crash on launch while 1.0.0 didn't

### ISSUE THREAD

**Issue by Un1q32** [Jonyboylovespie/LockedChests#2]
Mod list:
<img width="354" alt="Screenshot 2024-05-28 at 10 53 35 PM" src="https://github.com/Jonyboylovespie/LockedChests/assets/57970114/8511de72-c971-4c2a-a68b-8a4c47b75c07">

**Comment by Jonyboylovespie** [comment]
would appreciate if u could just send me a zip all of the mods, skipping the finding/downloading hastle.

**Comment by Un1q32** [comment]
[2024-05-28-6.log](https://github.com/Jonyboylovespie/LockedChests/files/15478183/2024-05-28-6.log)

probobly not a good idea to run random mods from a guy you dont know but whatever.
[mods.tar.gz](https://files.uniq.gay/mods.tar.gz)

**Comment by Jonyboylovespie** [comment]
just know, if you steal all my money, I will be sad

**Comment by Jonyboylovespie** [comment]
just in the log, I can see that the mixin used to check for locked chests isn't working, I've been making the mod on 1.20.4, so 1.20.6 could be the issue, I will investigate later

**Comment by Un1q32** [comment]
any updates on this?

**Comment by Jonyboylovespie** [comment]
I don’t think so, I’m probably switching to a plugin, as our Minecraft server now runs on paper

**Comment by Un1q32** [comment]
alright, im going to delete the mods.tar.gz file from my web server now then
